global.json

```diff
@@ -1,5 +1,6 @@
 {
     "sdk": {
-        "version": "6.0.400"
+        "version": "7.0",
```

Comment:
Is this a breaking change?

How will be for people using NET 6 and willing to get the latest updated versions of GeneticSharp?

Reply:
Yes, you would be required to upgrade to V7. However since they are very compatible, most people will upgrade to V7 anyway.
Feel free to ignore the PR though. Meanwhile I added some more tweaks for V7 anyway.